packages/in_app_purchase/in_app_purchase_android/android/src/test/java/io/flutter/plugins/inapppurchase/MethodCallHandlerTest.java

```diff
@@ -866,8 +1058,13 @@ public void isFutureSupported_false() {
   }
 
   private ArgumentCaptor<BillingClientStateListener> mockStartConnection() {
+    return mockStartConnection(BillingChoiceMode.PLAY_BILLING_ONLY);
+  }
+
+  private ArgumentCaptor<BillingClientStateListener> mockStartConnection(int billingChoiceMode) {
```

Comment:
Optional nit: Could you add a doc for this so it's easier to interpret why this may need to be called in some tests?

Reply:
Done